modules/videosipgw/JitsiVideoSIPGWSession.js

```diff
@@ -28,11 +28,16 @@ export default class JitsiVideoSIPGWSession {
      * @param {ChatRoom} chatRoom - The chat room this session is bound to.
      */
     constructor(sipAddress, displayName, chatRoom) {
+        super();
+
         this.sipAddress = sipAddress;
         this.displayName = displayName;
         this.chatRoom = chatRoom;
 
-        this.eventEmitter = new EventEmitter();
+        // The initial state is undefined. Initial state cannot be STATE_OFF,
+        // the session enters this state when it was in STATE_ON and was stopped
+        // and such session cannot be used anymore
```

Comment:
Probably the type should be documented with:
`/*`
`* The description...`
` * @type {Constants|undefined}`
`*/`
`this.state = undefined;`
Hmm actually I don't know how to document the type in case 'VideoSIPGWConstants' are imported as 'Constants'...